diff --git a/node_modules/proxyequal/lib/index.js b/node_modules/proxyequal/lib/index.js
index 9609399..d872ba1 100644
--- a/node_modules/proxyequal/lib/index.js
+++ b/node_modules/proxyequal/lib/index.js
@@ -3,7 +3,7 @@
 Object.defineProperty(exports, "__esModule", {
   value: true
 });
-exports.collectValuables = exports.collectShallows = exports.getProxyKey = exports.isProxyfied = exports.deproxify = exports.get = exports.proxyCompare = exports.proxyState = exports.proxyShallowEqual = exports.proxyShallow = exports.proxyEqual = exports.drainDifference = exports.sourceMutationsEnabled = exports.spreadGuardsEnabled = undefined;
+exports.collectValuables = exports.collectShallows = exports.getProxyKey = exports.isProxyfied = exports.deproxify = exports.get = exports.proxyCompare = exports.proxyState = exports.proxyShallowEqual = exports.proxyShallow = exports.proxyEqual = exports.sourceMutationsEnabled = exports.spreadGuardsEnabled = undefined;
 
 var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
 
@@ -242,6 +242,8 @@ var collectValuables = function collectValuables(lines) {
   }
   return lines.filter(function (line) {
     return values.indexOf(line) < 0;
+  }).map(function(line) {
+    return line.split('.');
   });
 };
 
@@ -249,6 +251,8 @@ var collectShallows = function collectShallows(lines) {
   var trie = (0, _searchTrie2.default)(lines);
   return lines.filter(function (value) {
     return trie(value + '.') || !value.lastIndexOf('.');
+  }).map(function(line) {
+    return line.split('.');
   });
 };
 
@@ -260,22 +264,12 @@ var get = function get(target, path) {
   return result;
 };
 
-var differs = [];
-
-var drainDifference = exports.drainDifference = function drainDifference() {
-  var d = differs;
-  differs = [];
-  return d;
-};
-
 var proxyCompare = function proxyCompare(a, b, locations) {
   for (var i = 0; i < locations.length; ++i) {
-    var key = locations[i];
-    var path = key.split('.');
+    var path = locations[i];
     var la = deepDeproxify(get(a, path));
     var lb = deepDeproxify(get(b, path));
     if (la !== lb) {
-      differs.push([key, 'differs', la, lb]);
       return false;
     }
   }
@@ -310,7 +304,6 @@ var proxyShallowEqual = function proxyShallowEqual(a, b, locations) {
   var pair = void 0;
   while (pair = tails.next().value) {
     if (!pair[1]) {
-      differs.push([pair[0], 'not equal']);
       return false;
     }
   }
